src/MaaWpfGui/ViewModels/UI/RecognizerViewModel.cs

```diff
@@ -550,9 +550,33 @@ public bool OperBoxParse(JObject details)
             List<Tuple<string, int>> operHave = new List<Tuple<string, int>>();
             List<Tuple<string, int>> operNotHave = new List<Tuple<string, int>>();
 
+            string localized_name;
+            switch (ConfigurationHelper.GetValue(ConfigurationKeys.Localization, string.Empty))
+            {
+                case "zh-cn":
+                    localized_name = "name";
+                    break;
+
+                case "ja-jp":
+                    localized_name = "name_jp";
+                    break;
+
+                case "ko-kr":
+                    localized_name = "name_kr";
+                    break;
+
+                case "zh-tw":
+                    localized_name = "name_tw";
+                    break;
+
+                default:
+                    localized_name = "name_en";
+                    break;
+            }
+
             foreach (JObject operBox in operBoxes.Cast<JObject>())
             {
-                var tuple = new Tuple<string, int>((string)operBox["name"], (int)operBox["rarity"]);
+                var tuple = new Tuple<string, int>((string)operBox[localized_name], (int)operBox["rarity"]);
```

Comment:
If a CN client (Arknights) player use MAA in English, when the new operator he has does not have an English name, maybe we can directly display the Chinese name of the operator.

Reply:
Discussion in https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/6726#discussion_r1349474985